dash/testing/application_runners.py

```diff
@@ -297,16 +308,40 @@ def start(self, app, start_timeout=2, cwd=None):
                     "dashr.run_server(app, cwd=os.path.dirname(__file__))"
                 )
 
+            # try copying all valid sub folders (i.e. assets) in cwd to tmp
+            # note that the R assets folder name can be any valid folder name
+            assets = [
+                os.path.join(cwd, _)
+                for _ in os.listdir(cwd)
+                if not _.startswith("__")
+                and os.path.isdir(os.path.join(cwd, _))
```

Comment:
🐄 why `_`? I prefer to use that only when the var isn't going to be used at all.

Reply:
I also like to use that in list comprehension especially when it gets more complicated. 
- shorter
- I think it's easier to read once you used to the convention, and can be more focus on the operation and logic side.

Reply:
Fair enough. I'd tend to use a 1-character name for that but I see your point.

Reply:
> Fair enough. I'd tend to use a 1-character name for that but I see your point.

1-char name is anti-pylint

Reply:
`_` isn't a 1-char name? 🤔

Reply:
btw, it's a valid 1-char name, certified.